tablo/interfaces/arcgis/views.py

```diff
@@ -661,3 +669,55 @@ def convert_wkt_to_esri_feature(response_items, for_layer):
                 features.append(feature)  # Some related fields have not been appended
 
     return features
+
+
+
+
+class ImageView(FeatureLayerView):
+
+
+    def handle_request(self, request, **kwargs):
+
+        from django.core.files.storage import default_storage as image_storage
```

Comment:
It is used on this line:

`       if image_storage.exists(s3_path):
            with image_storage.open(s3_path, 'rb') as fh:
`